HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -2656,13 +2659,28 @@ def distribution_system
     end
 
     def attached_cooling_system
-      return if distribution_system.nil?
+      return if distribution_system.nil? && @cooling_system_idref.nil?
+
+      # by heating system's idref
+      if not @cooling_system_idref.nil?
+        if @heating_system_type != HVACTypePTACHeating
+          fail "Heating system type: #{@heating_system_type} should not be attached to a cooling system. Only PTAC allowed."
```

Comment:
Can you add a test file and expected error message for this?

Reply:
I added a test file with furnace pointing to the central AC cooling system. This makes me wonder whether we should link heating and cooling systems in a more consistent way. So for now, PTAC cooling and heating is connected by the cooling system idref, but central ac and furnace are connected by distribution systems. Later we might allow ptac/pthp to be ducted (though from our previous conclusion it is rare, right?), in that case, if we retain the logic of using cooling system idref, will it look confusing if we don't allow furnace+ac to be connected in this way?